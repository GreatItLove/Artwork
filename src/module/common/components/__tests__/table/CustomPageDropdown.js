import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import CustomPageDropdown from '../../table/CustomPageDropdown';

describe('CustomPageDropdown', () => {
  it('Snapshot of CustomPageDropdown without props', () => {
    const wrapper = shallow(<CustomPageDropdown />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
  it('Snapshot of CustomPageDropdown with props', () => {
    const props = {
      maxPages: 10,
      currentPage: 2,
      setPage: jest.fn(),
      className: 'someClass',
      style: {}
    };
    const wrapper = shallow(<CustomPageDropdown {...props} />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
