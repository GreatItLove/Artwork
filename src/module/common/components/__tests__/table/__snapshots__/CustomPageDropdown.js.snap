// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomPageDropdown Snapshot of CustomPageDropdown with props 1`] = `
<div>
  <div
    className="someClass"
    style={Object {}}
  >
    <div
      className="item"
      onClick={[Function]}
      style={null}
    >
      1
    </div>
    <div
      className="item active"
      onClick={[Function]}
      style={
        Object {
          "pointerEvents": "none",
        }
      }
    >
      2
    </div>
    <div
      className="item"
      onClick={[Function]}
      style={null}
    >
      3
    </div>
    <div
      className="item"
      onClick={[Function]}
      style={null}
    >
      4
    </div>
    <div
      className="item"
      onClick={[Function]}
      style={null}
    >
      5
    </div>
    <div
      className="item"
      onClick={[Function]}
      style={null}
    >
      6
    </div>
    <div
      className="item"
      onClick={[Function]}
      style={null}
    >
      7
    </div>
    <div
      className="item"
      onClick={[Function]}
      style={null}
    >
      8
    </div>
    <div
      className="item"
      onClick={[Function]}
      style={null}
    >
      9
    </div>
    <div
      className="item"
      onClick={[Function]}
      style={null}
    >
      10
    </div>
  </div>
</div>
`;

exports[`CustomPageDropdown Snapshot of CustomPageDropdown without props 1`] = `
<div>
  <div>
    <div
      className="item"
      onClick={[Function]}
      style={null}
    >
      1
    </div>
  </div>
</div>
`;
